packages/hooks/src/useIsAccountStatusPresent.ts

```diff
@@ -48,7 +48,13 @@ const AccountStatusList = [
 
 type TAccountStatus = typeof AccountStatusList[number];
```

Comment:
can we somehow also extend types from API? 
GetAccountStatus['status'] from '@deriv/api-types'
there are no specific strings, but in case they will be added there.

Reply:
The type of status is `@deriv-api-types` is just string[]. The accepted values are written in docs.